tolumide-ng/AndelaBootcamp
# #163102460 Employing the Clean codes method

#### What does this PR do?
-Renames majority of the variables to enable easy understanding at first sight.
-Refactor some methods to enable adherence to clean codes
-Installation of code coverage for local testing
-Removes node_modules from the repository
-Removes, gitiignore from the repository

#### Description of Task to be completed?
-Have all endpoints working and easy to understand methods.

#### How should this be manually tested?
clone the repo, cd into it and run npm test to test the test coverage
or view the codes to see it's adherence to clean codes.

#### Any background context you want to provide?
There is the installation of some dependencies alongside this PR (mocha-lcov-reporter, nyc)

#### What are the relevant pivotal tracker stories?
#163102460
